front/src/modules/scenario/components/ScenarioLoader.tsx

```diff
@@ -8,8 +8,7 @@ type Props = {
 export default function ScenarioLoader({ msg }: Props) {
```

Comment:
Don't get the purpose of this component, as it duplicates, <Loader/> just to pass a single css class, I think it should be removed, and <Loader/> should be refactored, to handle the possibility to get custom class from the props

Reply:
Done !